### DISCUSSION TITLE
Releases and milestones

### DISCUSSION THREAD

**Discussion by famod** [quarkiverse/quarkus-cxf#339] [Ideas]
Hi all,

I find it hard to check what has changes in a release. As far as I am concerned (also as the maintainer of https://github.com/gitflow-incremental-builder/gitflow-incremental-builder), it's best practice use [milestones](https://github.com/quarkiverse/quarkus-cxf/milestones) to assign issues or PRs to an unpcoming release _and_ to [draft a release](https://github.com/quarkiverse/quarkus-cxf/releases) (either with a manual description or something generated).

I think it would help all users if `quarkus-cxf` could improve on that.

Cheers,
Falko

**Comment by shumonsharif** [comment]
Hey Falko,

Thanks for the suggestions.  I'm not sure how _draft releases_ (as opposed to regular releases) would fit into the existing Quarkiverse processes that we leverage, but will certainly take a look at whether it may be possible.

I believe we attempted to use milestones at some point - will certainly look into how that may work as well, though we may have a relatively low volume of issues and PRs going into each new release.

For now, here are some notes for the last release (I autogenerated it just now):
https://github.com/quarkiverse/quarkus-cxf/releases/tag/0.12.0

Shumon

**Reply by famod** [reply]
Thanks Shumon!

"Draft" might be an unfortunate term in this context but what I meant is just creating a release:
![screenshot](https://user-images.githubusercontent.com/22860528/147609799-1de965bb-ed5b-44c3-9a99-5101e69626f3.png)
